appinventor/appengine/src/com/google/appinventor/client/explorer/folder/ProjectFolder.java

```diff
@@ -245,7 +255,6 @@ public List<Project> getSelectedProjects() {
   public List<Project> getProjects() {
     return projects;
   }
-
```

Comment:
Not sure why this line was removed but it should be restored for code cleanliness.